conf/application.conf

```diff
@@ -70,6 +70,9 @@ play.http.router = prod.Routes
 # auditing (transaction monitoring) enabled.
 # The below controllers are the default exceptions to this rule.
 
+# this key is for local development only!
+queryParameter.encryption.key="gvBoGdgzqG1AarzF1LY0zQ=="
+
```

Comment:
We need to override this key with a blank value for base config so it does not work in qa/dev/prod environments. I want to make sure that we are not exposing our key for prod.